packages/core/mesh/teleport-extension-gossip/src/gossip.ts

```diff
@@ -108,7 +109,12 @@ export class Gossip {
         })
 
         .catch(async (err) => {
-          log.catch(err);
+          // TODO(nf): handle here or in the Extension?
+          if (err instanceof RpcClosedError) {
+            log.warn('sendAnnounce failed because of RpcClosedError', { err });
```

Comment:
I'd say we can even ignore this entirely. Gossip messaging does not guarntee reliability. This error just means that the connection was closed while we were waiting on an ACK for this message

Reply:
that's kind of what i was thinking as well, was just reluctant to totally swallow it as it's useful (at least now) for identifying other connection issues.

is DEBUG a worthwhile compromise?